core/embed/rust/src/micropython/obj.rs

```diff
@@ -289,6 +289,22 @@ impl TryFrom<&'static CStr> for Obj {
     }
 }
 
+impl TryFrom<(Obj, Obj)> for Obj {
+    type Error = Error;
+
+    fn try_from(val: (Obj, Obj)) -> Result<Self, Self::Error> {
+        // SAFETY:
+        //  - Should work with any micropython objects.
+        // EXCEPTION: Will raise if allocation fails.
+        let obj = catch_exception(|| unsafe { ffi::mp_obj_new_tuple(2, [val.0, val.1].as_ptr()) })?;
```

Comment:
Could we please put `[val.0, val.1]` into a variable?

Reply:
Sure, fixed in f17120e0d330692e053d902c01f648021702cec4.